### ISSUE TITLE
ESLint test

### ISSUE THREAD

**Issue by fabienrohrer** [omichel/webots-doc#773]
We would like to detect ESLint issues.

In addition to the test, `.eslintrc.json` should be duplicated in this repo, and eslint has to be integrated in travis.
Adding a test in webots that `.eslintrc.json` and `doc/.eslintrc.json` are identical would be even better.

References:
- https://eslint.org/docs/user-guide/getting-started
- https://gist.github.com/ronapelbaum/ef3c3459f01b002d0d719ca205401174

**Comment by omichel** [comment]
We should postpone the resolution of this issue until we implement the subrepo: https://github.com/omichel/webots/wiki/git-subrepo

**Comment by fabienrohrer** [comment]
It could indeed slightly help about the duplication of `.eslintrc.json`.

**Comment by fabienrohrer** [comment]
Indeed. Testing this in webots / Jenkins rather than webots-doc / Travis would be easier.
